_episodes/04-use-license.md

```diff
@@ -5,7 +5,185 @@ exercises: 0
+{: .callout}
+
+## Choose a licence for my code
+
+In most countries, the use of one's original work such as a software by others is protected by law. Because of that, it is important that creators choose an appropriate licence when sharing their code by email or in platforms such as GitHub, GitLab, Bitbucket. The definition of original work is not trivial, but as a rule of thumb most code that are dozens of lines long would probably fit as original work. If your code is too small, a dozen of characters, it can not classify as original work in which case it is not protected by any law.
+
+If you are the creator of an original work, you can decide the licence for it, but you should be careful of some common pitfalls:
+
+* you are contributing to an existent project, in this case your contribution should be licence compatible with the project that you are contributing to;
+* Giving away your copyright as part of your employment contract. This is very common and if this is the case you must seek permission of your employer to use a open source licence;
+* the terms and conditions of the grant funding the project, this is getting more common since funding agencies are requiring research outputs to be open access in which case you must choose a compatible licence.
+
+Since these are only three examples of issues that can arise, we recommend you to talk with the Intellectual Property (IP) office at your institution.
+
+Once added, a licence and all future versions of the software, will be under that licence. Most projects never need to change licenses, but occasionally circumstances change. When changing a project’s license, there is one fundamental thing to consider: **all** project's existing copyright holders should give their agreement in writing and signed. This is complicated because, without a previous copyright transfer agreement, the list of copyright holders can have thousands of names. We recommand to have a look at the [guide from GitHub on this question](https://opensource.guide/legal/#what-if-i-want-to-change-the-license-of-my-project) for more details.
```

Comment:
"a licence and all future versions [...]". Do you mean "the current version and all future versions [...]"?